src/pages/CalculatorPage.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable import/extensions */
+/* eslint-disable react/no-unescaped-entities */
+import React from 'react';
+import Calculator from '../components/Calculator.js';
```

Comment:
* Like the previous reviewer pointed out the error still persist 
![image](https://user-images.githubusercontent.com/52497246/129795172-95de8099-cbe2-4584-9ac6-7dec15b43831.png)
* Hint 💭 `../components/Calculator.js` calculator.js should be lowercase and not uppercase

Reply:
![image](https://user-images.githubusercontent.com/73256642/129796488-c97bc278-a80e-4105-bb15-51a2aed08b7c.png)
This is the file in my folder. I don't know why it keeps happening this issue to code reviewers.

Reply:
I can't reproduce that on my end.